README.md

```diff
@@ -148,7 +148,7 @@ Options:
 UnCSS can be used as a [PostCSS](https://github.com/postcss/postcss) Plugin.
 
 ```js
-postcss([ require('postcss-uncss').postcssPlugin ]);
+postcss([ require('postcss-uncss') ]);
```

Comment:
This was a mistake, however, the line should read:

    postcss([ require('uncss').postcssPlugin ]);